src/EditorFeatures/Test2/Diagnostics/AddImport/AddImportCrossLanguageTests.vb

```diff
@@ -302,6 +302,60 @@ namespace CSAssembly2
             Await TestMissing(input)
         End Function
 
+        <WpfFact>
+        Public Async Function AddProjectReference_CSharpToCSharp_ExtensionMethod() As Task
```

Comment:
Was missing a test for this scenario.  i found this because i originally changed recursion to *only* hit types, and not see members *at all*.  that didn't break anything, which i knew must be wrong.   added this, showed that it did indeed break.